config.md

```diff
@@ -115,7 +115,6 @@ For POSIX platforms the `mounts` structure has the following fields:
     },
     {
         "destination": "/data",
-        "type": "bind",
         "source": "/volumes/testing",
         "options": ["rbind","rw"]
```

Comment:
Does it makes sense to have rbind in the options if type:bind is removed

Reply:
Yes, `rbind`/`bind` are options and are not meant to be the "type". The type is not used when performing bind-mounts.

Reply:
@cyphar please help me understand this, if I remove rbind from my options it will then not do the binding right?

Reply:
> ... if I remove rbind from my options it will then not do the binding right?

You need either `bind` or `rbind` in `options`.  You do not need to set `type` for binds per the spec, and if/when opencontainers/runc#1753 lands, you won't need to set `type` for runc binds either.